docs/4.2/milv.config.yaml

```diff
@@ -29,6 +29,7 @@ white-list-external:
   - "https://teleport.example.com:8443"
   - "https://teleport.example.com"
   - "https://[route53_domain](#route53_domain"
+  - "https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-uhc-hmac-credentials-main"
```

Comment:
Why does this need whitelisting?

Reply:
It seems to be very 'flappy' / slow. We have done this perviously for docker links in 4.3 milv.